WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -230,38 +151,175 @@ extension UIColor {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .tertiaryLabel
+        } else {
+            return fallbackColor
+        }
+        #else
+        return fallbackColor
+        #endif
+    }
+
+    /// Very, very low contrast text
+    static var textQuaternary: UIColor {
+        let fallbackColor = UIColor.neutral(shade: .shade10)
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .quaternaryLabel
+        } else {
+            return fallbackColor
+        }
+        #else
+        return fallbackColor
+        #endif
+    }
+
+    static var textInverted = UIColor(light: .white, dark: .neutral(shade: .shade0))
+    static var textPlaceholder = neutral(shade: .shade30)
 
     /// Muriel/iOS navigation color
-    static var textInverted = UIColor.white
+    static var navigationBar = UIColor(light: .brand, dark: .neutral(shade: .shade0))
+
+    // MARK: -- Table Views
+    
+    static var divider: UIColor {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .separator
+        } else {
+            return muriel(color: .divider)
+        }
+        #else
+        return muriel(color: .divider)
+        #endif
+    }
 
-    static var navigationBar = muriel(color: .navigationBar)
+    /// WP color for table foregrounds (cells, etc)
+    static var tableForeground: UIColor {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .secondarySystemGroupedBackground
+        } else {
+            return .white
+        }
+        #else
+        return .white
+        #endif
+    }
 
-    static var tableBackground = muriel(color: .tableBackground)
+    static var tableForegroundUnread: UIColor {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .tertiarySystemGroupedBackground
+        } else {
+            return .primary(shade: .shade0)
+        }
+        #else
+        return .primary(shade: .shade0)
+        #endif
+    }
 
-    /// Muriel/iOS unselected color
-    static var unselected: UIColor {
-        if FeatureFlag.murielColors.enabled {
-            return muriel(color: MurielColor(name: .gray, shade: .shade20))
+    static var tableBackground: UIColor {
+        let fallbackColor = muriel(color: .tableBackground)
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .systemGroupedBackground
         } else {
-            return WPStyleGuide.greyLighten10()
+            return fallbackColor
         }
+        #else
+        return fallbackColor
+        #endif
     }
 
-    /// MARK: Muriel colors for buttons
+    /// For icons that are present in a table view, or similar list
+    static var listIcon: UIColor {
+        let fallbackColor = UIColor(light: .neutral(shade: .shade20), dark: .neutral(shade: .shade50))
+        #if XCODE11
+        if #available(iOS 13, *) {
+            return .secondaryLabel
+        } else {
+            return fallbackColor
```

Comment:
I have a doubt here: because for the dark mode it returns `.secondaryLabel` should the _fallbackColor_ be just `.neutral(shade: .shade20)`? Or the _listIcon_ should be just `UIColor(light: .neutral(shade: .shade20), dark: .neutral(shade: .shade50))`?